apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -272,6 +274,14 @@ def from_h5ad(
         platform_config: Platform-specific options used to create this array, provided in the form
         ``{"tiledb": {"create": {"sparse_nd_array_dim_zstd_level": 7}}}`` nested keys.
 
+        obs_id_name: Which AnnData ``obs`` column to use for append mode.
+
+        var_id_name: Which AnnData ``var`` column to use for append mode.
```

Comment:
Thanks for documenting these! A little more info on how they're used for appending might out help users. I think these args are also used to rename the index of incoming obs/var dataframes prior to ingest, right?